ddp-workspace/projects/ddp-sdk/src/lib/components/login/auth0-code-callback.component.ts

```diff
@@ -32,14 +32,20 @@ export class Auth0CodeCallbackComponent implements OnInit, OnDestroy {
         if (authCode) {
             this.log.logEvent(this.LOG_SOURCE, 'Will login to ' + this.configuration.localRegistrationUrl + ' using code ' + authCode);
 
-            const tempSession = this.session.isTemporarySession() ? this.session.session : null;
+            const value = sessionStorage.getItem('localAuthParams') || "{}";
```

Comment:
```suggestion
            const value = sessionStorage.getItem('localAuthParams') || '{}';
```

Reply:
And probably it would be better to make work with `sessionStorage` in separate method:
``` ts
private getLocalAuthParams(): object {
   const value = sessionStorage.getItem('localAuthParams') || '{}';
   sessionStorage.removeItem('localAuthParams');
   return value;
}
```

Reply:
How about `resetLocalAuthParams()`?  The side effect of clearing sessionStorage in a getter is a bit unusual.

Reply:
done